feat: add Zeebe tab button for empty tab

barmac opened the pull request:
Closes #2132

![image](https://user-images.githubusercontent.com/28307541/109769333-58653400-7bfa-11eb-9587-42303bad8b29.png)

pinussilvestrus commented on client/src/app/EmptyTab.js:
```diff
@@ -39,12 +39,14 @@ export default class EmptyTab extends PureComponent {
       <Tab className={ css.EmptyTab }>
         <p className="create-buttons">
           <span>Create a </span>
-          <button className="create-bpmn btn btn-secondary" onClick={ () => onAction('create-bpmn-diagram') }>BPMN diagram</button>
```
Seems like another component's tests depend on the `create-bpmn` selector: https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/__tests__/AppSpec.js#L265

barmac replied:
Sounds like an HoC snippet.

barmac replied:
Thanks for the hint.

andreasgeier commented:
I'm not sure if the row becomes too long when you apply the correct naming. Instead, I would suggest placing the buttons below each other like:

![Screen Shot 2021-03-04 at 18 47 02](https://user-images.githubusercontent.com/938244/110006490-061b3480-7d1a-11eb-8451-4031d25f3ea2.png)

I assume, at the latest when we introduce the form button we definitely need this layout change.

barmac commented:
Thanks Andi. I will add a commit regarding https://github.com/camunda/camunda-modeler/issues/2136 on top and this PR will include layout changes as well.

barmac commented:
For comparison:

![image](https://user-images.githubusercontent.com/28307541/110096509-41f7dd80-7d9e-11eb-837d-eefd7c31d24a.png)

or

![image](https://user-images.githubusercontent.com/28307541/110096547-4a501880-7d9e-11eb-8029-51564493e17f.png)

ingorichtsmeier commented:
Hello, what about the sequence of the buttons?

As it is unlikely to switch constantly between Zeebe and Camunda Platform, doesn't it look better if you group them by products?

* BPMN Diagram (Camunda Engine)
* DMN Diagram (Camunda Engine)
* BPMN Diagram (Zeebe Engine)

And use the same term for Camunda Engine. The DMN engine is rarely used by itself (from my perspective).

andreasgeier commented:
@barmac I would prefer the last proposal `Create a new file:` because creating a tab is not the goal of the user :) 

@ingorichtsmeier The naming and order above are what we agreed on in several rounds. I agree that we have here some not-ideal-solution as an intermediate step but we want to come back to an engine-agnostic start asap. This means, a future version should contain only `create BPMN / DMN / Form`

barmac commented:
I've adjusted the empty tab content and implemented https://github.com/camunda/camunda-modeler/issues/2136. 

The links should be active soon:

- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2132-empty-tab-button/camunda-modeler-2132-empty-tab-button-linux-x64.tar.gz
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2132-empty-tab-button/camunda-modeler-2132-empty-tab-button-mac.dmg
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2132-empty-tab-button/camunda-modeler-2132-empty-tab-button-mac.zip
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2132-empty-tab-button/camunda-modeler-2132-empty-tab-button-win-ia32.zip
- https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/2132-empty-tab-button/camunda-modeler-2132-empty-tab-button-win-x64.zip

andreasgeier reviewed (approved):
LGTM 👍

pinussilvestrus commented on client/src/app/EmptyTab.js:
```diff
@@ -37,20 +35,16 @@ export default class EmptyTab extends PureComponent {
           }
-        </p>
-
-        <Slot name="empty-tab-buttons" />
```
Fingers crossed there are no plugins using this slot.

But since we're not using it ourselves and didn't communicate this slot in the past, it's fine.

barmac replied:
This extension point was so secret that I didn't even know about it 🙈

pinussilvestrus reviewed (approved):
LGTM
